automation/check-patch.e2e-nmstate-functests.sh

```diff
@@ -1,71 +1 @@
 #!/usr/bin/env bash
-
-set -xeu
-
-# This script should be able to execute Kubernetes-nmstate
-# functional tests against Kubernetes cluster with
-# CNAO built with latest changes, on any
-# environment with basic dependencies listed in
-# check-patch.packages installed and docker running.
-#
-# yum -y install automation/check-patch.packages
-# automation/check-patch.e2e-nmstate-functests.sh
-
-teardown() {
```

Comment:
we also need to remove the [lane](https://github.com/kubevirt/project-infra/blob/main/github/ci/prow-deploy/files/jobs/kubevirt/cluster-network-addons-operator/cluster-network-addons-operator-presubmits.yaml#L198-L233) from project-infra

Reply:
We will do that at follow up since we cannot remove it at the same time.

Reply:
The idea was to convert the lane as noop on this PR and then after it merges we remove the prow job.